operator/pom.xml

```diff
@@ -174,6 +171,11 @@
                     <goals>
                         <goal>build</goal>
                     </goals>
+                    <configuration>
+                        <properties>
+                            <quarkus.package.filter-optional-dependencies>true</quarkus.package.filter-optional-dependencies> 
+                        </properties>
+                    </configuration>
```

Comment:
I think we don't need this any more.

Reply:
We don't but it's a good idea to leave it in as that's a more sensible default.